### PR TITLE
Update stablekoi adapter, add staking data

### PR DESCRIPTION
**Pull request by kintar2010**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/stablekoi.js exports TVL: 
        
 
 
    godwoken-staking          44.85 k
    godwoken                  6.67 M
    staking                   44.85 k
    
    total                    6.67 M

File: projects/stablekoi.js [hunk 1]
```diff
@@ -1,39 +1,59 @@
...
-  const balances = {}
-  const fixBalances = await getFixBalances(chain)
-  const tokensAndOwners = []
-  Object.keys(poolInfo).forEach(owner => {
-    poolInfo[owner].forEach(token => tokensAndOwners.push([token, owner]))
-  })
-  await sumTokens(balances, tokensAndOwners, chainBlocks[chain], chain)
-  return fixBalances(balances)
+async function tvl() {
+  const response = await axios.get("https://app.stablekoi.com/api/pools");
```
Comment (g1nt0ki) [inline projects/stablekoi.js:25]: Hi, sorry but I dont like the idea of replacing querying the chain with calling your api for data.
Like I said in your discord, I dislike the idea of hardcoded list that needs to be updated, can you create an api endpoint so we can fetch that?

Comment (kintar2010) [reply projects/stablekoi.js]: Updated

**Comment by llamatester** [comment]
The adapter at projects/stablekoi/index.js exports TVL: 
        
 
 
    godwoken-staking          52.27 k
    godwoken                  6.21 M
    staking                   52.27 k
    
    total                    6.21 M

**Pull request merged**